js/everything.js

```diff
@@ -364,56 +371,71 @@ function toggleElement(event, toggleID, buttonText) {
     }
 }
 
+/**
+ * wrapParseYouTubeVideoID the first v query value.
+ * Will return null if there's no v query param
+ * @return {string|null}
+ */
 function getCurrentVideoID() {
```

Comment:
If possible, I'd like to see this written with a single `return`. If it feels non-trivial we can do it later